daemons/based/based_callbacks.c

```diff
@@ -274,6 +275,28 @@ cib_common_callback_worker(uint32_t id, uint32_t flags, xmlNode * op_request,
                            pcmk__client_t *cib_client, gboolean privileged)
 {
     const char *op = crm_element_value(op_request, F_CIB_OPERATION);
+    int call_options = cib_none;
+
+    crm_element_value_int(op_request, F_CIB_CALLOPTS, &call_options);
+
+    if (pcmk_is_set(call_options, cib_transaction)) {
+        // Append request to client's transaction
+        const char *call_id = crm_element_value(op_request, F_CIB_CALLID);
+
+        int rc = based_extend_transaction(cib_client, op_request, privileged);
+
+        xmlNode *reply = create_cib_reply(op, call_id, cib_client->id,
+                                          call_options, rc, NULL);
+
+        if (rc != pcmk_rc_ok) {
+            crm_err("Could not extend transaction for client %s: %s",
+                    pcmk__client_name(cib_client), pcmk_rc_str(rc));
```

Comment:
I lean to warning for issues that only affect the client and are likely due to client input/behavior rather than the server failing at something

Reply:
Makes sense, will update